jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitRunner.kt

```diff
@@ -52,13 +52,15 @@ class SamInitRunner(
     }
 
     companion object {
-        private val expectedSamVersion = SemVer.parseFromText("0.6.0") ?: throw RuntimeException("SemVer parse error")
+        // TODO: change minimum to 0.7.0 before release
```

Comment:
Not related to the CR, we need to create an github issue to track this `TODO` in case we forget.